tests/utils.py

```diff
@@ -73,6 +70,16 @@ def wrapper(self, *args, **kwargs):
     return outer_wrapper
 
 @contextlib.contextmanager
+def assert_latest_log(log_action, node_key, index=-1):
+    node =  node_key
+    last_log = node.logs.latest()
+    node.reload()
+    yield
+    new_log = node.logs.order_by('-date')[-index - 1]
```

Comment:
Drive-by: Agreed, the `index` parameter is probably unnecessary; I don't believe we override the default value (`-1`) anywhere. So it should suffice to do:

```python
new_log = node.logs.latest()
```